common/src/main/scala/com/gu/emailservices/EmailService.scala

```diff
@@ -42,7 +40,8 @@ case class EmailFields(
     """.stripMargin
 }
 
-class EmailService(config: EmailConfig) extends StrictLogging {
+class EmailService(config: EmailConfig, executionContext: ExecutionContext) extends StrictLogging {
```

Comment:
You should be able to write:
`class EmailService(config: EmailConfig, implicit val executionContext: ExecutionContext)`
to avoid needing `ec` further down